.gitignore

```diff
@@ -1 +1,4 @@
-.idea
\ No newline at end of file
+.idea
+.vscode
+*__pycache__*
+.pylintrc
```

Comment:
Idea: we should use a common pylintrc or formatter guide.

Reply:
Oh yes, definitely :+1: